core/src/main/java/org/testcontainers/images/RemoteDockerImage.java

```diff
@@ -24,7 +24,7 @@
 @ToString
 public class RemoteDockerImage extends LazyFuture<String> {
 
-    public static final Set<DockerImageName> AVAILABLE_IMAGE_NAME_CACHE = new HashSet<>();
+    private static final Set<DockerImageName> AVAILABLE_IMAGE_NAME_CACHE = new HashSet<>();
```

Comment:
While I understand that the usage should not be public, we had this field defined as public, WDYT about deprecating it first?